docs/configuration.md

```diff
@@ -1,401 +1,406 @@
-
-Defaults to true
-
-
-### factorio.max_concurrent_commands
-
-Maximum number of commands being transmitted into the game in parallel.
-Since the rate at which long commands are streamed into the game is by
-default limeted to 100 bytes/tick and scales down to 25 bytes/tick when
-there's 20 players connected, long commands can hold up the command
-interface and cause large latencies in getting data pushed into the
-game. (The rates are controlled by the `segment_size` options in the
-server settings.)  To counteract this Clusterio sends up to this
-configured value number of commands in parallel over the RCON interface.
-This causes the game updates sent to clients to have this many command
-streams in parallel contained in them, drastically increasing the
-maxiumum size they can become.  Since commands are only processed so
-ofter the maximum rate of commands that can be sustained is roughly 30
-times this configured value per second, though note that large commands
-will lower this value.
-
-The game updates sent by the server is split into roughly 500 byte
-packets, and if any one of those parts are lost the entire update is
-resent.  This means a high number of concurrent commands can amplify
-resends due to packet loss and degrade the player experience up to the
-point of becomming completely unplayable.
-
-Prior to 0.18.7 the maximum practical size of game update sent where
-arround 8 kB due to the UDP receive buffer on windows defaulting to this
-value, larger updates would not get through the receive buffer and cause
-the connection to drop.  To stay below the 8 kB limit the maximum safe
-value for this option would be around 7000 / (3 * game_speed *
-maximum_segment_size), which is around 20 in normal circumstances.
-
-Defaults to 5
-
-
-### <plugin_name>.enabled
-
-Whether to enable the given plugin on the instance.  Note that plugins
-need to be enabled on the master server in order for them to be loaded
-on instances.
-
-Defaults to true
+Configuration
+=============
+
+
+Master Configuration
+--------------------
+
+### master.database_directory
+
+Directory used to store cluster data to.  Needs to be writeable by the
+master server.  This is also used by plugins to store its data.
+
+Defaults to "database".
+
+
+### master.http_port
+
+Port to host HTTP server on.  If set to null no HTTP server will be
+exposed.  At least one of this and master.https_port needs to be set to
+a port.
+
+Defaults to null.
+
+
+### master.https_port
+
+Port to host HTTPS server on.  Uses master.tls_certificate and
+master.tls_private_key as the certificate and private key for the HTTPS
+server.  If set to null no HTTPS server will be exposed.  At least one
+of this and master.http_port needs to be set to a port.
+
+Defaults to 8443.
+
+### master.address_bind
+Address to bind the master server on. If set to null the master server will be bound to 
+all interfaces. Has to be set to a valid interface. 
+
+Defaults to null.

```

Comment:
The spacings is not consistent with all of the other options in the file. There's should be two blank spaces before the headings and one blank space after it. There should also be two spaces after periods and the lines are wrapped to 72 characters.

I don't think the description here is saying the right thing. I suggest changing it to:
```
IP address to bind the HTTP and HTTPS ports on.  Useful to limit which
interface to accept connections from.  If set to null the unspecified
IPv6 address will be used, which on most systems means it will also
listen on the unspecified IPv4 address and accept connections from all
interfaces.
```

Reply:
:P that is better yes